openapi/workflow_execution_service.swagger.yaml

```diff
@@ -88,11 +88,27 @@ paths:
           in: query
           required: false
           type: string
-        - name: tag_search
+        - name: tag_key
           description: |-
             OPTIONAL
-            For each key, if the key's value is empty string then match workflows that are tagged with
-            this key regardless of value.
+            Filter returned workflow runs to only return those where the specified key is present in "tags".
+          in: query
+          required: false
+          type: string
+        - name: tag_value
+          description: |-
+            OPTIONAL
+            Filter workflow runs to only return those where this exact
+            key-value pair is present in "tags".  Only valid if
+            "tag_key" is also specified.  Filtering by "tag_value" only
+            is not supported.
+          in: query
+          required: false
+          type: string
+        - name: state
```

Comment:
Can the `state` parameter description just be replaced with:
```yaml
- $ref: '#/definitions/State'
```
...?

Reply:
Probably, since it only makes sense to accept a string in the state enum.